Monika After Story/game/script-topics.rpy

```diff
@@ -13046,3 +13046,150 @@ label mas_show_unseen:
+            m 3hua "Just ask me if you ever need help with anything!"
+
+            call monika_installed_python_version_menu
+            $ installed_py2 = _return
+
+            if installed_py2:
+                m 1hua "You'll do amazing things, I just know it!"
+
+            else:
+                m 3rksdlb "So we can't really help each other that much..."
+                m 1hub "But that's okay!"
+                m 1eub "I hope you have fun with your project."
+
+        "To help you come closer to our reality.":
+            m 1sub "Really?"
+            m 1hubfb "Yay!"
+            m 1ekbfa "Thank you so much, [player]."
+            m 3ekbfa "It means so much that you're willing to do that for me."
+            m 3hubfb "I'd be happy to teach you the basics of Python!"
+            m 1rksdlb "Wait..."
+
+            call monika_installed_python_code_level_menu
+
+            if persistent._mas_pm_can_code_python:
+                m 2hub "Oh! That's great!"
+                m 3rksdla "I'm sorry I made assumptions about your skill level."
+
+            else:
+                m 3hksdlb "That's okay!"
+                m 1eka "Again, I'm happy to teach you everything I know."
+                call monika_installed_python_version_menu
+
+                $ installed_py2 = _return
+
+                if installed_py2:
+                    m 1hua "We'll do amazing things together, I just know it!"
+                    m 1ekbfb "Thank you again, [player]."
+
+                else:
+                    m 1ekbfb "I really appreciate the gesture, though."
+                    m 3hubfb "You're so sweet! <3"
+
+        "I don't know.":
+            m 1hub "Ehehe!"
+            m 1eka "That's okay, [player]."
+            m 3eub "Until you know what you want to program, I can teach you how to use Python!"
+            m 1rka "That is, if you have the same version of Python as I do..."
+
+            call monika_installed_python_version_menu
+            $ installed_py2 = _return
+
+            if installed_py2:
+                m 3rksdlb "Also, [player]?"
+                call monika_assume_code_level
+
+            else:
+                m 1lksdlb "Well, that's okay..."
+                m 1eka "I hope you have fun, anyway."
+                m 1rsc "..."
+                m 1esd "Hey, [player]?"
+                call monika_assume_code_level
+
+        "To help a friend.":
+            m 3eub "That's great!"
+            m 1eka "You're so selfless, [player]."
+            m 1hub "I can't wait to see what you create!"
+
+            call monika_installed_python_version_menu
+            $ installed_py2 = _return
+
+            if installed_py2:
+                m 1hua "Maybe both you and your friend could help me out once you know how to code!"
+                m 1hksdlb "Wait..."
+                call monika_installed_python_code_level_menu
+
+            else:
+                m 3rksdlb "So we can't really help each other that much..."
```

Comment:
Might have to parameterize the label for python version. The response for using py3 doesn't really fit here anymore

Reply:
Alright